src/components/organisms/AdvancedTable/index.tsx

```diff
@@ -73,22 +59,28 @@ function AdvancedTable<T extends BaseObject>(props: AdvancedTableProps<T>) {
   );
 
   return (
-    <Styles>
+    <Box m="2rem" minW="90%">
       {props.showTopBar && <TopBar name={props.name} length={data.length} />}
       {props.showToolbar && (
         <Toolbar
           showButton={props.showButton}
```

Comment:
What is the point of prop drilling `showButton` if it is not being passed already from any of the parent components (`<CategoriesTable />`, `<DishesTable />`, `<IngredientsTable />`, `<ProductsTable />`)?

Reply:
During initial implementation (couple weeks ago) I wanted to make this components more generic, but as you already pointed - we always pass `true` as default prop.
I removed unnecessary props from those implementations as you suggested.